bigquery/samples/utils.py

```diff
@@ -1,66 +0,0 @@
-# Copyright 2015, Google, Inc.
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-
-# [START get_service]
-def get_service():
-    """returns an initialized and authorized bigquery client"""
-
-    from googleapiclient.discovery import build
-    from oauth2client.client import GoogleCredentials
-
-    credentials = GoogleCredentials.get_application_default()
-
-    # Create and return the service object for v2 of the bigquery api. You can
-    # browse available apis and versions here:
-    #   https://developers.google.com/api-client-library/python/apis/
-    #
-    # The build() method also takes care of injecting the necessary scopes
-    # to access the given service, when using application default credentials.
-    return build('bigquery', 'v2', credentials=credentials)
-# [END get_service]
-
-
-# [START poll_job]
-def poll_job(service, projectId, jobId, interval=5.0, num_retries=5):
```

Comment:
It will, we'll need to update any docs that reference ``utils.py`` to point to the actual thing. @jerjou volunteered to do that.